### PR TITLE
Fix for Store 1179, Highlight the tab in the publisher tabs ribbon  

### PR DESCRIPTION
**Pull request by tmkasun**
In this PR:
* Change the background color of the tab in publisher asset details page , To indication which tab currently user in
* Add a new string object named `currentPage` to  `store.publisher` JS object in publisher.

This PR resolve the issue described [here] (https://wso2.org/jira/browse/STORE-1179)

### CODE REVIEW

File: apps/publisher/themes/default/js/popover.js [hunk 1]
```diff
@@ -43,6 +43,33 @@ var popoverFunction = (function(){
     return returnObj;
 })();
 
+var currentPageFunction = (function(){
```
Comment (daneshk) [inline apps/publisher/themes/default/js/popover.js]: formatting issue

File: apps/publisher/themes/default/js/popover.js [hunk 2]
```diff
@@ -43,6 +43,33 @@ var popoverFunction = (function(){
...
+    var returnObj = {};
+    returnObj.init = function(){
+        console.log("tmkasun debug yhehaa "+store.publisher.currentPage);
```
Comment (daneshk) [inline apps/publisher/themes/default/js/popover.js]: why console.log here ????
& formatting issue.

File: apps/publisher/themes/default/js/popover.js (hunk 1 above)
Comment (tmkasun) [inline apps/publisher/themes/default/js/popover.js]: Thanks, I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/38e71195e4ea971e38601e6537597a13e9c24459)

File: apps/publisher/themes/default/js/popover.js (hunk 2 above)
Comment (tmkasun) [inline apps/publisher/themes/default/js/popover.js]: Thanks, This has been fixed with [this commit](https://github.com/wso2/carbon-store/commit/63493eef6a6b9acc6f9c1381a39e2434a81a90c6)

**Pull request merged**